test/suites/container_devices_disk.sh

```diff
@@ -59,3 +61,43 @@ test_container_devices_disk_shift() {
   lxc storage volume delete "${POOL}" foo-shift
   lxc stop foo -f
 }
+
+test_container_devices_disk_ceph() {
+  local LXD_BACKEND
+
+  LXD_BACKEND=$(storage_backend "$LXD_DIR")
+  if ! [ "${LXD_BACKEND}" = "ceph" ]; then
+    return
+  fi
+  RBD_POOL_NAME=lxdtest-$(basename "${LXD_DIR}")-disk
+  ceph osd pool create $RBD_POOL_NAME 1
+  rbd create --pool $RBD_POOL_NAME --size 50MB
+  rbd map --pool $RBD_POOL_NAME --name admin
+  RBD_POOL_PATH="/dev/rbd/${RBD_POOL_NAME}"
+  mkfs.ext4 -m0 $RBD_POOL_PATH
+  rbd unmap $RBD_POOL_PATH
+  lxc launch testimage ceph-disk -c security.privileged=true
+  lxc config device add ceph-disk rbd disk source=ceph:$RBD_POOL_NAME/my-volume ceph.user_name=admin ceph.cluster_name=ceph path=/ceph
```

Comment:
You'll want `${RBD_POOL_NAME}` instead of `$RBD_POOL_NAME` or shellcheck may complain